lib/cli/commands/add.js

```diff
@@ -0,0 +1,84 @@
+// Use
+const addCommand = {
+	command: "add [--development] [--optional] <framework-libraries..>",
+	describe: "Add a SAPUI5/OpenUI5 framework library to the project configuration.",
+	middlewares: [require("../middlewares/base.js")]
+};
+
+addCommand.builder = function(cli) {
+	return cli.positional("framework-libraries", {
+		describe: "Framework library names",
+		type: "string"
+	}).option("development", {
+		describe: "Add as development dependency",
+		alias: "D",
```

Comment:
Why uppercase? For the other commands, the shortcut has the same case as the option, as far as I can see. 

I find consistency within our cli more appealing than consistency of a single command with yarn. Or can we have two aliases?

Reply:
Will this feature make it into the 2.0 release?

Reply:
@codeworrior  - it is not only yarn - moreover its also consistency with npm:
https://docs.npmjs.com/cli/install

So, i personally like the abbreviation with the uppercase alias. 👍

Reply:
Yes, we looked at the aliases of the `npm install` command here. However, I would personally prefer a `--dev` and `--opt` alias. We can add as many alias as we like 😏

> Will this feature make it into the 2.0 release?

Yes, definitely. Our documentation already mentions it: https://github.com/SAP/ui5-tooling/pull/221/files#diff-1a523bd9fa0dbf998008b37579210e12R32

Reply:
Great, thanks! So we just add all of them.

Reply:
I'll discuss our alias options with @matz3 in the morning

Reply:
Looking into yarn, they call --opt => --optional -> I would try to align myself as much as possible with already existing CLIs. This helps people to easily use the one or the other. The uppercase aliases are common across npm and yarn and that is why I am also in favor to have them.